packages/metrics/src/model/mutant-model.ts

```diff
@@ -32,6 +32,9 @@ export class MutantModel implements MutantResult {
   public killedByTests: TestModel[] | undefined;
   public sourceFile: FileUnderTestModel | undefined;
 
+  #_coveredByTests: Map<string, TestModel> = new Map();
```

Comment:
I used an underscore as to not clash with the `coveredByTests` property. I didn't want to change that property to a map, since I wanted to keep the behavior the same.

But I agree that this could do without the `_`, so it might need a different name.